src/blocks/venue/render.php

```diff
@@ -20,17 +20,24 @@
 	return;
 }
 
-$gatherpress_venue_information = json_decode( get_post_meta( $gatherpress_venue->ID, '_venue_information', true ) );
+// phpcs:ignore
+$gp_venue_map = ( get_post( $gatherpress_venue->ID )->post_content ?: '' );
 
-Utility::render_template(
-	sprintf( '%s/build/blocks/venue-information/render.php', GATHERPRESS_CORE_PATH ),
-	array(
-		'attributes' => array(
-			'name'        => $gatherpress_venue->post_title,
-			'fullAddress' => $gatherpress_venue_information->fullAddress ?? '', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
-			'phoneNumber' => $gatherpress_venue_information->phoneNumber ?? '', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
-			'website'     => $gatherpress_venue_information->website ?? '',
-		),
-	),
-	true
```

Comment:
Let's keep this as it is since venue post could have more information than is needed in block. Just add the map embed. Could also add a link back to the venue post for additional information.